addon/components/horizontal-list-swiper/gesture/styles.less

```diff
@@ -21,7 +18,10 @@
     overflow:hidden;
   }
   .asp-scroll-area {
-    transition: all .8s ease-in-out;
+    transition-property: all;
+    transition-timing-function: ease-in-out;
+    transition-delay: 0s;
+    transition-duration: inherit;
```

Comment:
Is there a reason you separated these? The syntax order for shorthand is:
_property name | duration | timing function | delay_
so this would be 
```
.asp-scroll-area {
  transition: all inherit ease-in-out 0s;
}
```

Reply:
@NikkiAlgarCentrica yes, because the `duration` property is provided by the parent component https://github.com/britishgas-engineering/ember-appointment-slots-pickers/tree/feature/transfer-slot-pickers-to-open-source/addon/components/horizontal-list-swiper/no-delay-on-transitions-in-test